CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing 
+This is a document outlining how to contribute to the Tūhura Tech wiki.
+
+There are two main ways of contributing:  
+- Suggesting - this involves creating a new [issue on Github](https://github.com/Tuhura-Tech/Wiki/issues) and informing us of a way to improve the wiki. Alturnatively, you can notify the team directly on our [Discord server](https://github.com/Tuhura-Tech/Wiki/issues)
+- Directly - to do this you need to [fork our repository](https://github.com/Tuhura-Tech/Wiki/fork) and editing it, then creating a pull request to merge with the main site
+
+## Issues and suggestions
+If you have a suggestion or an issue you have found with the wiki please go to our [Github issues](https://github.com/Tuhura-Tech/Wiki/issues) page and start a new issue
+
+### Creating an issue
+For a new issue on the [Github issues](https://github.com/Tuhura-Tech/Wiki/issues) issues page, press 'New issue'   
+From there decide which issue you would like to raise:
+- Docs Request - this is for if you have information that you would like to be documented on the Tūhura Tech wiki, or you have a request for something you would like us to look into and document
+- Report a security vulnerability - if you have seen a security vulnerability, let us know privately here
+
+### Documentation request
```

Comment:
Is it worth putting this info in the Issue Template instead of here? We can just link to the open issue button but all this info should be there.

Reply:
It's not there, if you look at the documentation request it seems to just be the default instead of information about documentation

Reply:
Yeah, I think we should put just *Title*, *Short description*, and *Detailed/additional information* in the issue template. 

But at the bottom of the issue template it says to refer to our guide and there's a link to this file so I think we should have this guide for writing here since it goes into more detail and just have the titles in the issue template. 

I will change the link though